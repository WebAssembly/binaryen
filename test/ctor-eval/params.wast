(module
  (func "test1" (param $x i32)
    ;; The presence of params stops us from evalling this function, at least
    ;; not with --ignore-external-input (see ignore-external-input.wast)
    (nop)
  )
)
