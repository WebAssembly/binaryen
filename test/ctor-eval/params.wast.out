(module
 (type $i32_=>_none (func (param i32)))
 (export "test1" (func $0))
 (func $0 (type $i32_=>_none) (param $x i32)
  (nop)
 )
)
